cmd/skaffold/app/cmd/config/set.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"reflect"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	yaml "gopkg.in/yaml.v2"
+)
+
+func NewCmdSet(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set",
+		Short: "Set a value in the global skaffold config",
+		Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
There is a builtin arg cobra arg validator here you can use

Reply:
not a huge fan of the error message it gives, but I'll use it

```
FATA[0000] accepts 2 arg(s), received 3 
```